### ISSUE TITLE
Support For Vectorising Packets

### ISSUE THREAD

**Issue by iBuyMountainDew** [bluewolf2027/freeedit#3]
This feature would greatly enhance your application's usability; vectorising packets of all bytecode variations would be an incredible advantage over VSC and Atom.

**Comment by milse113** [comment]
Absolutely; Might I also suggest implementing support for 5G UPF aswell?
